### PR TITLE
Add correct version number to script and correctly ignore when linting

### PR DESCRIPTION
**Pull request by taymoork2**
## Description

Provide the correct version number when generating the iife script and have been ignored when CI runs the linter through it, as it was causing Jenkins jobs to fail during static analysis.

Fixes # (not reported)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -448,7 +449,7 @@ ansiColor('xterm') {
 
                   // Rebuild with correct version number
                   sh 'npm run build'
-                  sh 'npm run build:script'
+                  sh "npm run build:script -- --ver=${version}"
```
Comment (andrewholsted) [inline Jenkinsfile]: is `version` here the same as the sdk package.json version?

File: rollup.config.js [hunk 2]
```diff
@@ -9,34 +10,35 @@ import license from 'rollup-plugin-license';
...
-    commonjs(),
-    globals(),
-    builtins(),
-    terser(),
-    license({
-      banner: `/*! Cisco Spark SDK v${version} */`
-    })
-  ]
-};
+export default function ({versionNumber}) {
```
Comment (andrewholsted) [inline rollup.config.js]: How is this method called? Seems like we would need to pass it somewhere from the build script through to here?

File: Jenkinsfile [hunk 3]
```diff
@@ -432,7 +432,8 @@ ansiColor('xterm') {
                   version = version.trim()
                   echo "next version is ${version}"
                   sh 'npm run build'
-                  sh 'npm run build:script'
```
Comment (andrewholsted) [inline Jenkinsfile]: Changing from build to build:script makes me nervous

File: Jenkinsfile (hunk 1 above)
Comment (taymoork2) [reply Jenkinsfile]: yes

File: rollup.config.js (hunk 2 above)
Comment (taymoork2) [reply rollup.config.js]: gets called with rollup `npm run build:script -- --ver=${version}`

File: Jenkinsfile (hunk 3 above)
Comment (taymoork2) [reply Jenkinsfile]: build still builds the packages
build:script is a different and separate process

**Pull request merged**